client/test/e2e/cash/cash.spec.js

```diff
@@ -65,17 +65,18 @@ describe('Cash Payments Module', function () {
     it('navigating directly to /cash should be re-routed to selected cashbox after a selection is made', function () {
 
       // our target is cashboxB
-      var target = path.concat('/' + cashboxB.id);
+      var target = path.concat('/' + cashboxA.id);
 
-      // implicitly choose cashbox B by navigating to it directly
+      // implicitly choose cashbox B by navigating to it directly 
```

Comment:
Hmm. 

"Our target is cashboxB" --> cashboxA.id seems illogical to me.  What do I not understand?